pages/enterprise/setup.rst

```diff
@@ -154,10 +301,10 @@ Your plugin directory should look similar to this after installing the enterpris
 ::
 
   plugin/
-  ├── graylog-plugin-aws-3.3.2.jar
-  ├── graylog-plugin-collector-3.3.2.jar
-  ├── graylog-plugin-enterprise-3.3.2.jar
-  └── graylog-plugin-threatintel-3.3.2.jar
+  ├── graylog-plugin-aws-3.3.8.jar
+  ├── graylog-plugin-collector-3.3.8.jar
+  ├── graylog-plugin-enterprise-3.3.8.jar
+  └── graylog-plugin-threatintel-3.3.8.jar
```

Comment:
is this still true?

Reply:
I think so. I only updated the version here. It's just an example of what a plugins directory might look like.

Reply:
In the past we tried to keep that in the same picture what customers can see on their systems. that is why I ask.